lte/protos/session_manager.proto

```diff
@@ -103,8 +103,9 @@ message LocalEndSessionResponse {
 message UpdateTunnelIdsRequest {
   SubscriberID sid = 1;
   uint32 bearer_id = 2;
-  string enb_teid = 3;
-  string agw_teid = 4;
+  uint32 enb_teid = 5;
+  uint32 agw_teid = 6;
+  reserved 3,4;
```

Comment:
is there a reason why we do not simply use 3 and 4 for these fields, there is no legacy for this feature.

Reply:
Yeah changing type is unsafe from string to uint but I guess as we don't use this anywhere yet we can ignore that